tests/php/sync/test_class.jetpack-sync-posts.php

```diff
@@ -426,6 +425,7 @@ public function test_sync_update_post() {
 	private function reset_sync() {
 
 		Jetpack_Sync::$actions   = array();
+		Jetpack_Sync::$client->sync_queue = array();
```

Comment:
I would rather we invoked a method to reset the state of the client than directly assigned an array to this property.